osquery/tables/system/darwin/disk_encryption.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ *  Copyright (c) 2015, Sharvil Shah
```

Comment:
Also note that this will get code modded to include: Copyright (c) 2014, Facebook, Inc. as per the Facebook CLA.